VSRAD.Package/DebugVisualizer/VisualizerHeaderControl.xaml

```diff
@@ -21,18 +23,186 @@
+            <Style x:Key="NDRangeInput" TargetType="local:NumberInput">
+                <Setter Property="IsEnabled" Value="{Binding GroupIndexEditable}"/>
+                <Style.Triggers>
+                    <Trigger Property="Validation.HasError" Value="true">
+                        <Setter Property="ToolTip" Value="{Binding Path=(Validation.Errors)/ErrorContent, RelativeSource={RelativeSource Self}}"/>
+                    </Trigger>
+                </Style.Triggers>
+            </Style>
+            <Style x:Key="NDRange3DInput" TargetType="local:NumberInput" BasedOn="{StaticResource NDRangeInput}">
+                <Setter Property="Visibility" Value="{Binding Options.VisualizerOptions.NDRange3D, Converter={StaticResource BoolToVisibility}}"/>
+            </Style>
+            <ControlTemplate x:Key="ComboBoxToggleButton" TargetType="{x:Type ToggleButton}">
+                <Grid>
+                    <Grid.ColumnDefinitions>
+                        <ColumnDefinition />
+                        <ColumnDefinition Width="20" />
+                    </Grid.ColumnDefinitions>
+                    <Border x:Name="Border"
+                            Grid.ColumnSpan="2"
+                            CornerRadius="0"
+                            Background="{DynamicResource {x:Static vsshell:VsBrushes.WindowKey}}"
+                            BorderBrush="{DynamicResource {x:Static vsshell:VsBrushes.ToolWindowBorderKey}}"
+                            BorderThickness="1" />
+                    <Border Grid.Column="0"
+                            CornerRadius="0"
+                            Margin="1"
+                            Background="{DynamicResource {x:Static vsshell:VsBrushes.WindowKey}}"
+                            BorderBrush="{DynamicResource {x:Static vsshell:VsBrushes.ToolWindowBorderKey}}"
+                            BorderThickness="0,0,1,0" />
+                    <Path x:Name="Arrow"
+                          Grid.Column="1"
+                          Fill="{DynamicResource {x:Static vsshell:VsBrushes.WindowTextKey}}"
+                          HorizontalAlignment="Center"
+                          VerticalAlignment="Center"
+                          Data="M0,0 L0,2 L4,6 L8,2 L8,0 L4,4 z"/>
+                </Grid>
+            </ControlTemplate>
+
+            <ControlTemplate TargetType="{x:Type ComboBoxItem}" x:Key="ComboBoxItemTemplate">
+                <StackPanel Orientation="Horizontal" x:Name="SP"
+                                Background="{DynamicResource {x:Static vsshell:VsBrushes.WindowKey}}">
+                    <Button Background="{DynamicResource {x:Static vsshell:VsBrushes.WindowKey}}"
+                            Width="22" Height="22"
+                            Command="{Binding PinnedButtonCommand, ElementName=Root}" CommandParameter="{Binding}"
+                            Foreground="Black" RenderTransformOrigin="0.5 0.5" BorderThickness="0">
```

Comment:
Probably not the best color for some VS themes like Dark. It's better to use `{DynamicResource {x:Static vsshell:VsBrushes.WindowTextKey}}` instead.

Please also note that `OnHover` color may need to be changed.

Reply:
Thanks, fixed in 5da6922